beater/nvidiagpubeat.go

```diff
@@ -68,6 +68,18 @@ func (bt *Nvidiagpubeat) Run(b *beat.Beat) error {
 		os.Setenv("PATH", paths.Paths.Home+";"+os.Getenv("PATH"))
 	}
 	ticker := time.NewTicker(bt.config.Period)
+
+	// Construct an instance of Query and for each type in the array add it into a map
+	query := nvidia.NewQuery()
```

Comment:
Is it slightly nicer to pass `bt.config.System` and `bt.config.GPU` into the constructor?